doc/api/deprecations.md

```diff
@@ -950,11 +950,21 @@ deprecated if the assigned value is not a string, boolean, or number. In the
 future, such assignment may result in a thrown error. Please convert the
 property to a string before assigning it to `process.env`.
 
+<a id="DEP00XX"></a>
+### DEP00XX: crypto.Decipher.finaltol
```

Comment:
Should `Decipher` here and below be lowercased since it's referring to a prototype method and not a static method?

Reply:
I agree that `decipher.finaltol` is better than `Decipher.finaltol` here, but `crypto.decipher.finaltol` looks weird (since there is no `decipher` property on `crypto`). I changed all occurrences to lowercase and removed the `crypto.` prefix. Is there a better way to do this?

Reply:
I think what dropping `crypto.` and lowercase is fine, especially since `.final()` is linked to and should give enough context.